src/index.js

```diff
@@ -473,14 +514,25 @@ class ServerlessThundraPlugin {
                     }
                 })
                 .catch(err => {
-                    const rethrownError = new Error('Error while getting Thundra layers');
-                    rethrownError.stack = rethrownError.stack.split('\n').slice(0, 2).join('\n') + '\n' + err.stack;
-                    reject(rethrownError);
+                    const rethrownError = new Error(
+                        'Error while getting Thundra layers'
+                    )
+                    rethrownError.stack =
+                        rethrownError.stack
+                            .split('\n')
+                            .slice(0, 2)
+                            .join('\n') +
+                        '\n' +
+                        err.stack
+                    reject(rethrownError)
                 })
         })
     }
 
     getLatestLayerVersion(runtime, region) {
+        if (runtime == 'dotnetcore') {
+            runtime = 'dotnetcore2.1'
```

Comment:
Do we also support .NET Core 3.1 runtime by `dotnetcore2.1` layer?

Reply:
Yes, we support other .NET Core runtimes as custom runtimes, but we can not officially say that it is possible to all cases. We need to perform more tests.